src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1568,6 +1560,26 @@ public ResponseEntity<DynamicSimulationStatus> getDynamicSimulationStatus(@Param
                 ResponseEntity.noContent().build();
     }
 
+    @GetMapping(value = "/studies/{studyUuid}/security-analysis/parameters")
+    @Operation(summary = "Get security analysis parameters on study")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The security analysis parameters")})
+    public ResponseEntity<SecurityAnalysisParametersValues> getSecurityAnalysisParametersValues(
+            @PathVariable("studyUuid") UUID studyUuid) {
+        return ResponseEntity.ok().body(studyService.getSecurityAnalysisParametersValues(studyUuid));
+    }
+
+    @PostMapping(value = "/studies/{studyUuid}/security-analysis/parameters")
+    @Operation(summary = "set security analysis parameters on study, reset to default ones if empty body")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The security analysis parameters are set")})
+    public ResponseEntity<Void> setSecurityAnalysisParametersValues(
+            @PathVariable("studyUuid") UUID studyUuid,
+            @RequestBody(required = false) SecurityAnalysisParametersValues securityAnalysisParametersValues,
+            @RequestHeader(HEADER_USER_ID) String userId) {
+        studyService.setSecurityAnalysisParametersValues(studyUuid, securityAnalysisParametersValues, userId);
+        return ResponseEntity.ok().build();
+    }
+
+
```

Comment:
remove this empty new line

Reply:
It's validated by CheckStyle so no worries.

Reply:
Done